cpp/src/copying/contiguous_split.cu

```diff
@@ -195,10 +195,15 @@ __device__ void copy_buffer(uint8_t* __restrict__ dst,
       std::size_t idx = (num_bytes - remainder) / 4;
       uint32_t v = remainder > 0 ? (reinterpret_cast<uint32_t const*>(src)[idx] - value_shift) : 0;
       while (remainder) {
-        uint32_t const next = bit_shift > 0 || remainder > 4
+        // if we're doing a validity copy, do we need to read an extra bitmask word to OR it's
+        // relevant bits in?
```

Comment:
Is this a question or are you describing the purpose of `have_extra_rows`?

Reply:
Describing the purpose :)   I'll clean up the wording.

Reply:
Describing the purpose of `have_extra_rows` :)   I'll clean up the wording.